integrations.yml

```diff
@@ -11,6 +11,13 @@
 # 8 - Contracts and Proposals
 # 9 - Developer-only Tools
 #
+
+-
+  name: "BrightGauge Software"
+  description: "Track data about Basecamp usage through dashboards, gauges and reports, alongside other integrations, including Harvest and Dropbox."
+  website: "https://www.brightgauge.com/"
+  icon: "brightgauge.png"
+  category: 3
```

Comment:
Thanks @Madelene! Would you mind moving this to the end of the file instead of having it here at the top?